src/foam/nanos/dig/DigWebAgent.java

```diff
@@ -213,6 +213,65 @@ public void execute(X x) {
           outputException(x, resp, format, out, error);
 
           return;
+        } else if (Format.JSONJ == format ) {
+          String dataJson = "[";
+          String dataJsonJ[] = data.split("\\r?\\n");
+          for (String i:dataJsonJ){
+            i = i.trim();
+            dataJson += i.substring(2, i.length()-1) + ',';
```

Comment:
Ok, i preferred the test for 'p'
`         if (i.startsWith("p(") && i.endsWith(")")) {`
as that ensures we only put 'p' operations, the blind substring can let through 'r' operations on command 'put'.  
Is this really what the JDAO does?